lib/purchases_flutter.dart

```diff
@@ -802,7 +804,10 @@ class Purchases {
   static void handleLogHandlerEvent(MethodCall call) {
     final args = Map<String, dynamic>.from(call.arguments);
     final logLevelName = args['logLevel'];
-    final logLevel = LogLevel.values.firstWhere((e) => e.name.toUpperCase() == logLevelName, orElse: () => LogLevel.info);
+    final logLevel = LogLevel.values.firstWhere(
+      (e) => describeEnum(e).toUpperCase() == logLevelName,
+      orElse: () => LogLevel.info,
+    );
```

Comment:
Fixes without changing the lower end of the sdk
```
warning • This API is available since SDK 2.15.0, but constraints '>=2.12.0 <3.0.0' don't guarantee it • lib/purchases_flutter.dart:418:60 • sdk_version_since
```